examples/create_groups.ipynb

```diff
@@ -1,149 +1,599 @@
+    "\n",
+    "### Packages\n",
+    "\n",
+    "Import the libraries used in this notebook:"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [],
    "source": [
+    "import numpy as np\n",
     "import tiledb\n",
-    "from tiledb.cf import Group, GroupSchema, VirtualGroup\n",
-    "import numpy as np"
+    "from tiledb.cf import Group, GroupSchema\n",
+    "from mpl_toolkits.mplot3d import Axes3D\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create numpy arrays\n",
+    "\n",
+    "Variables to set the size of the arrays:"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "t_size = 365\n",
+    "dense_size = 100\n",
+    "sparse_size = 2000"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Functions used to create a dataset:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def ripple(x,y,t):\n",
+    "    return (np.sin(t*(x**2+y**2))/t)\n",
+    "\n",
+    "def ripple2(x,y,z,t):\n",
+    "    return (np.sin(t*(x**2+y**2))/t)+z"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Data for the dense arrays"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/usr/local/Caskroom/miniconda/base/envs/netcdf-dev/lib/python3.7/site-packages/ipykernel_launcher.py:2: RuntimeWarning: invalid value encountered in true_divide\n",
+      "  \n"
+     ]
+    }
+   ],
+   "source": [
+    "Dx_values = np.arange(dense_size)+1\n",
+    "Dy_values = np.arange(dense_size)+1\n",
+    "Dt_values = np.arange(t_size)+1\n",
+    "\n",
+    "D_xyt_values = \\\n",
+    "    np.fromfunction(lambda Dx_values, Dy_values, Dt_values: ripple(Dx_values, Dy_values, Dt_values), \n",
+    "                    (dense_size,dense_size,t_size))\n",
+    "\n",
+    "D_xy_values = np.nanmean(D_xyt_values, axis=2)\n",
+    "D_t_values = np.mean(D_xyt_values, axis=(0, 1))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Data for the sparse arrays"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Sx_values = np.random.randint(dense_size, size=(sparse_size))+1\n",
+    "Sy_values = np.random.randint(dense_size, size=(sparse_size))+1\n",
+    "Sz_values = np.random.randint(dense_size, size=(sparse_size))+1\n",
+    "St_values = np.random.randint(t_size, size=(sparse_size))+1\n",
+    "\n",
+    "S_xyzt_values = ripple2(Sx_values,Sy_values,Sz_values,St_values)"
```

Comment:
```suggestion
    "Sx_values = np.random.randint(1, dense_size + 1, size=(sparse_size))\n",
    "Sy_values = np.random.randint(1, dense_size + 1, size=(sparse_size))\n",
    "Sz_values = np.random.randint(1, dense_size + 1, size=(sparse_size))\n",
    "St_values = np.random.randint(t_size, size=(sparse_size))\n",
    "\n",
    "S_xyzt_values = ripple2(Sx_values,Sy_values,Sz_values,St_values)"
```
This should do the same thing (create random numbers in the range `[1, dense_size + 1)`.